src/react-demo/Grid.es6.js

```diff
@@ -90,7 +90,7 @@ function GridPage () {
             <tr>
               <td>direction</td>
               <td><code>column</code></td>
-              <td>Sets direction of the grid to column. Default is 'row'.</td>
+              <td>Sets direction of the grid to column. Default is &quot;row&quot;.</td>
```

Comment:
I feel like it will show `&quot;` instead of `'` in the preview. Could you please take a screenshot of how it looks?

Reply:
![image](https://user-images.githubusercontent.com/35793666/177470629-034c35e4-edb2-48f6-bc2c-e535930ea175.png)

Reply:
I changed it to `&apos;` because I realised that the quotes are double quoted.

Reply:
The code blocks show `&apos;` but i think that should be fine right?
![image](https://user-images.githubusercontent.com/35793666/177471324-f8f6853e-f6d1-4eab-8d30-0d8b1c2347ba.png)

Reply:
Yeah I think it is ok for code blocks!